src/components/ProfileForm/ProfileForm.js

```diff
@@ -0,0 +1,164 @@
+import React, { useState } from "react";
+import {
+  Alert,
+  Button,
+  Col,
+  Container,
+  Form,
+  InputGroup,
+  Row,
+} from "react-bootstrap";
```

Comment:
See if the import works like this:
`import Alert from "react-bootstrap/Alert";`

If yes, then use that way, this will reduce bundle size. It's called `tree shaking import`
If not, then ignore.